src/styles/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 div:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
1. Checklist p1 (don't use tags, use classes);
2. Checklist p4 - keep your block 'stars' operational in case you change block modifier, fix it please
https://prnt.sc/iTyoA9h_zq63

Reply:
Fixed, thank you!